evalml/pipelines/components/estimators/classifiers/lightgbm_classifier.py

```diff
@@ -107,9 +107,10 @@ def fit(self, X, y=None):
     def predict(self, X):
         X_encoded = self._encode_categories(X)
         predictions = super().predict(X_encoded)
-        if self._label_encoder:
-            predictions = pd.Series(self._label_encoder.inverse_transform(predictions.astype(np.int64)))
-        return predictions
+        if not self._label_encoder:
```

Comment:
Reorganized this way so that we don't convert to series if not necessary.

Reply:
Yupperino, `super().predict(X_encoded)` should return a Woodwork object, so we can return it directly. Otherwise, we need to get the series to do some magic, before converting it back to WW (blegh).